### ISSUE TITLE
Forge 4.1.80 reverts resource pack changes

### ISSUE THREAD

**Issue by Renari** [Pokecube-Development/Pokecube-Issues-and-Wiki#1039]
#### Issue Description:
It looks like Forge 4.18.0 reverts the changes they made to cache resource loading in resource packs, which breaks Pokecube.

These changes were made in 870e1b4b48762efb410136d2650416841a666d51 however looking at that commit, it seems some other changes were also made in that commit, so simply reverting the commit isn't sufficient.

Deleting `info.init(PackType.SERVER_DATA);` will allow the mod to build, however I believe that fix leaves a bunch of superfluous code.

#### What happens:
The mod will fail to load on Forge 4.1.80.

#### What you expected to happen:
The mod will load on forge 4.1.80

#### Steps to reproduce:

1. Install Forge 4.1.80 with Pokecube AIO

____
#### Affected Versions (Do *not* use "latest"): Replace with a list of all mods you have in.

- Pokecube AIO: 3.17.12
- Minecraft: 1.18.2
- Forge: 4.1.80

**Comment by Thutmose** [comment]
this is a duplicate of #1012 See my reasons posted there for not working on 40.1.80. Essentially it makes dev testing of the mod take nearly an order of magnitude longer to load worlds, so I will not revert those changes until forge actually adds content in a patch (ie if 40.1.81 adds something useful, I will update and revert the change then, but not until that)
